CHANGELOG.md

```diff
@@ -17,6 +17,9 @@
 -   [86cuhfyvp] Implemented functionality of Trip listing card using dummy data
 -   [86cugqgpt] Implemented filters and sort for action bar
 -   [86cueubmz] Implemented trip booking page and booking details form UI
+    <<<<<<<<< Temporary merge branch 1
```

Comment:
conflcits?

Reply:
Rectified all conflicts in commit : https://github.com/nijin-26/bus-ticket-booking-react/pull/24/commits/c24d59b2ac3714c62037845ee9aef7d83179401e